lib/modules/manager/fvm/index.ts

```diff
@@ -6,6 +6,6 @@ export { extractPackageFile } from './extract';
 export const supportedDatasources = [FlutterVersionDatasource.id];
 
 export const defaultConfig = {
-  fileMatch: ['(^|/)\\.fvm/fvm_config\\.json$'],
+  fileMatch: ['(^|/)\\.fvm(?:/fvm_config\\.json|rc)$'],
```

Comment:
Split this in two different regexes for better readability:
```suggestion
  fileMatch: [
    '(^|/)\\.fvm/fvm_config\\.json$',
    '(^|/)\\.fvmrc$'
   ],
```